test/Http.test.ts

```diff
@@ -0,0 +1,25 @@
+import { HTTP } from '@src/services/Http';
+import { expect } from 'chai';
+import { stub } from 'sinon';
+import axios from 'axios';
+
+describe('Http', () => {
+	it('Get method timeouts after given time', async () => {
+		// Arrange:
+		stub(axios, 'get').returns(
+			new Promise((_, reject) => {
+				setTimeout(() => {
+					reject(Error('timeout'));
+				}, 5_000);
+			}),
+		);
+
+		// Act:
+		try {
+			await HTTP.get('SOME_FAKE_URL', { timeout: 2_000 });
```

Comment:
because only expect is in catch, if this statement doesn't throw, test will pass, which probably isn't what you want

Reply:
Fixed, instead of try-catch, used promised based assertion 👍